README.md

```diff
@@ -1 +1,3 @@
 # gha-test
```

Comment:
After checking the change request, you need to manually dismiss this review to get it merged